backend/src/tasks/alerts.py

```
@@ -0,0 +1,387 @@
+                    'success': True,
+                    'alerts_deleted': len(old_alerts),
+                    'cutoff_date': cutoff_date.isoformat(),
+                    'timestamp': datetime.now().isoformat()
+                }
+
+        result = self.run_async(cleanup_alerts)
+        logger.info(f"Alert cleanup completed: {result}")
+        return result
+
+    except Exception as exc:
+        logger.error(f"Alert cleanup task failed: {str(exc)}", exc_info=True)
```

Ruff (0.14.4)

Use explicit conversion flag

Replace with conversion flag

(RUF010)